app/assets/javascripts/sephora_style_guide/alert.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function(){
```

Comment:
There is an alerts.js that you can just copy-paste https://github.com/twbs/bootstrap/blob/v4-dev/js/dist/alert.js

Reply:
Gave it a go but it seems like an overkill and doesn't have the functions that we for our style guide need i.e. timeout and hide (instead of DOM removal)